AdobeStockAsset/Model/GetAssetById.php

```diff
@@ -64,7 +64,7 @@ public function execute(int $adobeId): Document
         $items = $this->getAssetList->execute($searchCriteria)->getItems();
         if (empty($items) || 1 < count($items)) {
             $message = __('Requested image doesn\'t exists');
-            throw new NoSuchEntityException($message);
+            throw new NotFoundException($message);
```

Comment:
@sivaschenko  I put NotFoundException because the @api interface declares it. Should I change it to the NoSuchEntityException in both implementation and interface? https://github.com/magento/adobe-stock-integration/blob/1.1-develop/AdobeStockAssetApi/Api/GetAssetByIdInterface.php#L24